src/app/modules/main/chat_section/chat_content/users/controller.nim

```diff
@@ -124,8 +124,12 @@ proc getChat*(self: Controller): ChatDto =
 proc getChatMembers*(self: Controller): seq[ChatMember] =
   if self.belongsToCommunity:
     let myCommunity = self.getMyCommunity()
-    return myCommunity.getCommunityChat(self.chatId).members
-  else:
+    # TODO: when a new channel is added, chat may arrive earlier and we have no up to date community yet
+    # see log here: https://github.com/status-im/status-desktop/issues/14442#issuecomment-2120756598
+    # should be resolved in https://github.com/status-im/status-desktop/issues/14219
+    let members = myCommunity.getCommunityChat(self.chatId).members
+    if members.len > 0:
+      return members
     return self.chatService.getChatById(self.chatId).members
```

Comment:
In theory, since you changed status-go to return the members in the "normal" chat, I think the code above to get from the community is no longer useful, but I think it's safer to wait for the refactor just in case.

Reply:
It should be ok for new chats, but old ones will be empty until new `CommunityDescription` arrives